src/app/legacy/psammead/psammead-story-promo/src/index.test.jsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import { arrayOf, bool, shape, string, oneOf } from 'prop-types';
 import MediaIndicator from '#psammead/psammead-media-indicator/src';
-import LiveLabel from '#psammead/psammead-live-label/src';
 import styled from '@emotion/styled';
 import { GEL_SPACING_HLF } from '#psammead/gel-foundations/src/spacings';
 import { render } from '../../../../components/react-testing-library-with-providers';
 import latin from '../../../../components/ThemeProvider/fontScripts/latin';
+import LiveLabel from '../../../../components/LiveLabel';
```

Comment:
Let's update the props being sent to the LiveLabel component here (e.g. remove `dir`)